### PR TITLE
Colm/ssttp 1149/check amounts you owe page

### PR DESCRIPTION
**Pull request by cavster**
[https://jira.tools.tax.service.gov.uk/browse/SSTTP-1149](https://jira.tools.tax.service.gov.uk/browse/SSTTP-1149)

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -173,12 +173,13 @@ ssttp.calculator.misalignment-page.subtitle=The figures you entered don’t matc
...
 ssttp.calculator.misalignment-page.account-breakdown.title=View a breakdown
-ssttp.calculator.misalignment-page.account-breakdown.total=TOTAL:
+ssttp.calculator.misalignment-page.account-breakdown.total=Total:
 ssttp.calculator.misalignment-page.possible-reasons.title=These differences could be because:
-ssttp.calculator.misalignment-page.possible-reasons.list.1=you haven’t filed your Self Assessment tax return
-ssttp.calculator.misalignment-page.possible-reasons.list.2=you filed less than 3 days ago and your account hasn’t updated yet
-ssttp.calculator.misalignment-page.possible-reasons.list.3=you didn’t include all outstanding amounts in your calculation
-ssttp.calculator.misalignment-page.possible-reasons.list.4=you included an amount that isn't related to your Self Assessment
+ssttp.calculator.misalignment-page.possible-reasons.list.1=didn’t include all outstanding amounts in your calculation
+ssttp.calculator.misalignment-page.possible-reasons.list.2=included an amount that isn't related to your Self Assessment
```
Comment (Smudgey) [inline conf/messages]: The apostrophe is missing on the page, need to double it '' to escape the apostrophe, or copy the one used in the line above

File: conf/messages [hunk 2]
```diff
@@ -173,12 +173,13 @@ ssttp.calculator.misalignment-page.subtitle=The figures you entered don’t matc
...
+ssttp.calculator.misalignment-page.possible-reasons.list.3=haven’t filed your Self Assessment tax return
+ssttp.calculator.misalignment-page.possible-reasons.list.4=filed less than 3 days ago and your account hasn’t updated yet
+ssttp.calculator.misalignment-page.advice=Recalculate your instalments using the total in your tax account. If you can't continue call us on 0300 200 3822.
```
Comment (Smudgey) [inline conf/messages]: Same as above for can't

File: conf/messages [hunk 3]
```diff
@@ -401,6 +402,7 @@ ssttp.unavailable.number=0300 200 3822
 
 #global errors
 global.error.400.title=Bad request=400
+global.error.400.title=Bad request=400
```
Comment (Smudgey) [inline conf/messages]: unintended duplication?

**Review by Smudgey — CHANGES_REQUESTED** [review]

File: conf/messages [hunk 4]
```diff
@@ -175,11 +175,11 @@ ssttp.calculator.misalignment-page.total-in-account=Your tax account shows:
 ssttp.calculator.misalignment-page.account-breakdown.title=View a breakdown
 ssttp.calculator.misalignment-page.account-breakdown.total=Total:
 ssttp.calculator.misalignment-page.possible-reasons.title=These differences could be because:
-ssttp.calculator.misalignment-page.possible-reasons.list.1=didn’t include all outstanding amounts in your calculation
+ssttp.calculator.misalignment-page.possible-reasons.list.1=didn'’t include all outstanding amounts in your calculation
 ssttp.calculator.misalignment-page.possible-reasons.list.2=included an amount that isn't related to your Self Assessment
```
Comment (nkaraolis) [inline conf/messages]: The apostrophe for "isn't" doesn't appear on the page so probably needs a double apostrophe

**Review by nkaraolis — CHANGES_REQUESTED** [review]

**Review by Smudgey — APPROVED** [review]

**Pull request merged**